lua/starfall/libs_cl/midi.lua

```diff
@@ -0,0 +1,88 @@
+if not SF.Require("midi") then return function() end end
+
+local checkluatype = SF.CheckLuaType
+
+-- Midi Library
```

Comment:
Didn't notice in my earlier review, docs are incorrect for this file, each one needs to start with three dashes at the top or else they won't be detected.